### ISSUE TITLE
How to add HTML to ui.table?

### ISSUE THREAD

**Issue by tjakobi** [zauberzeug/nicegui#217]
Hi and thank you for your work on nicegui!

I tried to add HTML to `ui.table` and found this documentation based on JustPy https://justpy.io/grids_tutorial/adding_html/. 

However, I realized that since nicegui 1.0, JustPy has been phased out, making me wonder what the easiest way would be to add HTML to a complete table or specific columns (like JustPY did).

Thank you,

Tobias

**Comment by falkoschindler** [comment]
Hi Tobias, thank for your feedback!

So far we haven't thought about rendering HTML in table cells. But, fortunately, it isn't hard to add this feature in NiceGUI 1.0. In [b80cb0c](https://github.com/zauberzeug/nicegui/commit/b80cb0cd170d8ac1bf1b834c3a031aacb2ba9740) I introduce an optional `html_columns` parameter for `ui.table` to allow specifying column indices that should be rendered as HTML:

```python
table = ui.table({
    'columnDefs': [{'field': 'name'}, {'field': 'age'}],
    'rowData': [{'name': '<strong>Alice</strong>', 'age': 18}],
}, html_columns=[0]).classes('max-h-40')
```

The new parameter will be available in version 1.0.10.

**Comment by tjakobi** [comment]
Works beautifully with the new 1.0.10 release, thank you very much!
